### ISSUE TITLE
W tensorflow/core/grappler/utils/graph_view.cc:830] No registered 'swish_f32' OpKernel for GPU devices compatible with node {{node model_1/swish_38/swish_f32}}

### ISSUE THREAD

**Issue by bitwangdan** [google/automl#24]
This error occurred when I was training on the GPU，Has anyone else experienced this problem?

**Comment by KaqiKaqi** [comment]
same warning.

**Comment by bitwangdan** [comment]
@KaqiKaqi have you solved it?

**Comment by mingxingtan** [comment]
Hmm, I am surprised that swish doesn't work on GPU. If so, could you replace:

https://github.com/google/automl/blob/e90a471a0ebedd9162afa1971dbc494ba12d42c7/efficientdet/utils.py#L30

with "relu_fn = x * tf.sigmoid(x)"?  (they are equivalent but tf.nn.swish is supposed to be more memory efficient)

**Comment by mingxingtan** [comment]
As shown in this [tutorial](https://github.com/google/automl/blob/master/efficientdet/tutorial.ipynb), it seems to run well with GPU. closing this issue for now.

**Comment by Cocoalate** [comment]
HI, i'm facing the same problem and replace tf.nn.swish with tf.sigmoid works for me. I suppose it's happenning because I'm using tf.compat.v1 to train main.py under tf2.0 environment
